docs/book/deployment.md

```diff
@@ -389,6 +390,34 @@ that will be used to call Lambda's [updateFunctionConfiguration](
 You can use this to modify the Lambda's timeout, change the memory size, or
 add your own environment variables. All sub keys are optional.
 
+#### DynamoDB TTL (optional)
+
+Litexa has limited support for setting a [Time To Live (TTL)](
+  https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html)
+field with each database entry. This may be useful for limiting your table
+data to active users by automatically deleting old user data. To set a Time To
+Live specification, add the `dynamoDbConfiguration` object to your Litexa config:
+
+```json
+{
+  "deployments": {
+    "development": {
+      "module": "@litexa/deploy-aws",
+      "askProfile": "suncoast",
+      "awsProfile": "prototyping",
+      "dynamoDbConfiguration": {
+        "ttlName": "timeToLive", // the name of the TTL attribute to set
+        "ttlInSeconds": 2592000 // how many days (in seconds) the new/updated record lives for
```

Comment:
Should match the official properties, to minimize confusion. Maybe something like:

```js
dynamoDbConfiguration: {
  TimeToLiveDescription: {
    "AttributeName": "ttl", // needs to match the attribute name specified when enabling TTL
    "ExpirationTime": 2592000 // specified in seconds (would wipe data every 30 days)
  }
}
```

Would also change the MD language to js so the comments don't break.

Reply:
Purposely made them not the same because `ttlInSeconds`is a configuration, whereas ExpirationTime is a value. I think it (at least, the uppercasing format) differentiates more clearly between what is a dynamo configuration and what is Litexa-specific. What do you think?

Reply:
Agreed: How about `secondsToLive`, to avoid the "ttl" redundancy if you nest it below a `ttl` property?